task_templates/3_pipelines/5_python3_sklearn_binary/create_pipeline.py

```diff
@@ -105,7 +109,8 @@ def to_string(x):
     transformers=[
         ("num", numeric_pipeline, numeric_selector),
         ("cat", categorical_pipeline, categorical_selector),
-        ("txt", text_pipeline, text_selector),
+        # comment out because of TfidfVectorizer
```

Comment:
remove this and instead have a comment explaining that this was removed in this PR